### DISCUSSION TITLE
trouble with wakeup.Command

### DISCUSSION THREAD

**Discussion by foxharp** [languitar/autosuspend#259] [Q&A] [answered]
I'm trying to get my machine to wake up at a specific time every day, in order to coincide with a server which will attempt to perform a backup of the machine at that time.

First:  Is there a way to do this directly?  It's similar to "periodic", but different:  i.e., "every day at 3am".

As I understand it, the command option for wakeup.Command should return the usual UNIX "seconds since epoch" format.  So I'd like to simply do something like:
```
[wakeup.Command]
enabled=true
command = date -d '3am tomorrow' '+%s'
```
But of course this will fail if the machine sleeps at, say, 2am.  It won't wake in 1 hour, it will wake in 25 hours.

I suspect I'm asking for a common use case, so likely there's an easy way to do it.

Thanks.

**Comment by languitar** [comment, marked as answer]
This is comparable to the use case described in this FAQ entry: https://autosuspend.readthedocs.io/en/v4.2.0/faq.html#how-do-i-keep-a-system-active-at-daytime

Create a local ical file with the times you want the system to ve active and configure the ical checks to use that file.

**Reply by foxharp** [reply]
Thank you.  I guess when I read the man page, it never occurred to me that I'd want to use the ical support, since I don't use ical anywhere else in my life.

Also:  perhaps it would be useful to include a link to readthedocs.io in the man page? I found autosuspend via the Ubuntu package manager, and had never seen that documentation before you directed me there.  Something like this would have been very helpful!  "Full documentation for autosuspend can be found at https://autosuspend.readthedocs.io/, including frequently asked questions."

In any case, many thanks for the pointer, and for autosuspend itself.  It will replace a kludgy shell script I wrote which does some, but not all, of the same functions.

**Reply by languitar** [reply]
Thanks for the suggestion. The documentation link has been added: https://github.com/languitar/autosuspend/commit/2b4007cb9dfb5c4702cdf1f17e800b1dcaa142a7

If the ical check fulfills your use case, could you please mark this as the answer.
